### PR TITLE
Added Swedish language and some link fixes for #18.

### PR DESCRIPTION
**Pull request by Lazylllama**
Changed the Ticket link to discord invite because if you were not in the discord it would say you dont have permission to access it. Changed up the link to work with #18 so the swedish language wasnt even inteded to be in this PR but my stupid ass did the translation on the main branch 🤦🏻 Also remove the locale and country from the paypal link so it uses their locale instead. Please lmk if something needs to be changed.

### CODE REVIEW

File: ValorantStoreChecker/Commons/Constants.swift [hunk 1]
```diff
@@ -32,16 +32,16 @@ struct Constants{
...
+        static var sourceCode = "https://github.com/SoloUnity/valorant-store-checker-iOS"
         static var website = "https://valorantstore.net"
         static var discord = "https://discord.gg/vK5mzjvqYM"
         static var review = "https://apps.apple.com/app/id1637273546?action=write-review"
-        static var donate = "https://paypal.me/SoloUnityNA1?country.x=CA&locale.x=en_US"
+        static var donate = "https://paypal.me/SoloUnityNA1"
         static var privacy = "https://www.craft.do/s/fQxdg6aSyp8WAk"
         static var appStore = "https://apps.apple.com/ca/app/store-checker-for-valorant/id1637273546"
-        static var translation = "https://github.com/SoloUnity/Valorant-Store-Checker-App#help-translate"
+        static var translation = "https://github.com/SoloUnity/Valorant-Store-Checker-App#-help-translate"
```
Comment (SoloUnity) [inline ValorantStoreChecker/Commons/Constants.swift:44]: Link no longer works

**Review by SoloUnity — CHANGES_REQUESTED** [review]

File: ValorantStoreChecker/Commons/Constants.swift (hunk 1 above)
Comment (SoloUnity) [reply ValorantStoreChecker/Commons/Constants.swift:44]: The link doesnt work, other than that all good!

**Comment by SoloUnity** [comment]
My exams are coming up so this pull request will most likely only be out at the end of next month as I'd like to add other new features to the update as well.

File: ValorantStoreChecker/Commons/Constants.swift (hunk 1 above)
Comment (Lazylllama) [reply ValorantStoreChecker/Commons/Constants.swift:44]: It needs to have the **-** in order to work with  #18 cause the readme has emojis.

**Review by SoloUnity — APPROVED** [review]

**Pull request merged**